packages/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -30,14 +30,14 @@ import * as mongodb from 'mongodb';
 import * as shimmer from 'shimmer';
 
 /** MongoDBCore instrumentation plugin for OpenTelemetry */
-export class MongoDBCorePlugin extends BasePlugin<typeof mongodb> {
+export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
   private readonly _SERVER_METHODS = ['insert', 'update', 'remove', 'command'];
   private readonly _CURSOR_METHODS = ['_next', 'next'];
 
-  private readonly _COMPONENT = 'mongodb-core';
+  private readonly _COMPONENT = 'mongodb-nodejs';
   private readonly _DB_TYPE = 'mongodb';
 
-  readonly supportedVersions = ['>=2 <3'];
+  readonly supportedVersions = ['>=2 <4'];
```

Comment:
do we not support 3?

Reply:
Yes, starting from 2 and under 4, so 3 is covered

Reply:
i see I misread